src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/InvestPreviewModal.vue

```diff
@@ -67,7 +69,7 @@ const amountMap = computed(
   (): AmountMap => {
     const amountMap = {};
     fullAmounts.value.forEach((amount, i) => {
-      if (hasAmount(i)) amountMap[props.tokenAddresses[i]] = amount;
+      amountMap[props.tokenAddresses[i]] = amount;
```

Comment:
There is no amount check anymore

Reply:
Pon wanted to show tokens - even when the amount is `0`

Reply:
Ahh right, this is the preview modal, got it.